protocol/x/clob/keeper/msg_server_create_clob_pair.go

```diff
@@ -14,7 +17,15 @@ import (
 func (k msgServer) CreateClobPair(
 	goCtx context.Context,
 	msg *types.MsgCreateClobPair,
-) (*types.MsgCreateClobPairResponse, error) {
+) (resp *types.MsgCreateClobPairResponse, err error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	defer func() {
+		if err != nil {
+			errorlib.LogErrorWithBlockHeight(ctx.Logger(), err, ctx.BlockHeight(), metrics.DeliverTx)
```

Comment:
Can you pass the keepers logger? Specifically using [this](https://github.com/dydxprotocol/v4-chain/blob/46ce587c6a713c469050e462cea7b4322b082b2d/protocol/x/clob/keeper/keeper.go#L133).

Reply:
ah I see, my bad. Will update